gemrb/plugins/SDLAudio/SDLAudio.h

```diff
@@ -79,7 +80,7 @@ class SDLAudio : public Audio {
 	void clearBufferCache();
 	Mix_Chunk* loadSound(const char *ResRef, unsigned int &time_length);
 
-	int XPos, YPos;
+	int listenerXPos, listenerYPos;
```

Comment:
You can init them here instead, now that we're on c++11.

Reply:
also, since you're changing it anyway, maybe make it a `Point` instead